src/core/config/defaults.go

```diff
@@ -71,7 +71,7 @@ var (
 			BulkSize:           20,
 			ReportInterval:     1 * time.Minute,
 			CollectionInterval: 15 * time.Second,
-			Mode:               "aggregation",
```

Comment:
this change seems to be unnecessary

Reply:
In the code the keyword "aggregated", I thought only to bring the defaults into line with the code?